src/explore-education-statistics-frontend/src/modules/table-tool/components/FixedMultiHeaderDataTable.tsx

```diff
@@ -11,7 +11,7 @@ interface Props {
   columnHeaders: string[][];
   rowHeaders: string[][];
   rows: string[][];
-  footnotes?: TableData['footnotes'];
+  footnotes: TableData['footnotes'];
```

Comment:
This can still be optional as long as a default value for `footnotes` is assigned